script/index.ts

```diff
@@ -31,6 +31,14 @@ async function main() {
     log(`Running in dry-run mode. The script won't push and create PRs.`, {
       color: 'yellow',
     })
+  } else if (!CONFIG.dryRun && CONFIG.githubToken === undefined) {
+    log(
+      'No Github token found. Branches will be created, but all pull requests will fail.',
+      {
+        type: 'warn',
+        color: 'yellow',
+      }
+    )
   }
```

Comment:
I think we should stop the script execution whatsoever, because even if it creates the branches, another run would trigger the same processes.

```suggestion
  } else if (CONFIG.githubToken == null) {
    log(
      'No Github token found. Make sure to define it in your "config.ts" file',
      {
        type: 'error',
        color: 'red',
      }
    )
    process.exit(1)
  }
```

Reply:
Good point, I would still keep the dry run verification though. In my opinion, github token shouldn't matter if someone is making a dry run (no remote branches will be created after all). What do you think?

Reply:
@kaisermann, pushed your suggestion, but changed
`  } else if (CONFIG.githubToken == null) {`
to
`  } else if (!CONFIG.dryRun && CONFIG.githubToken == null) {`